openapi/index.yaml

```diff
@@ -67,6 +67,35 @@ paths:
           description: No service found for the provided ID.
         "429":
           description: Too many requests
+  "/services/{service_id}/latest":
```

Comment:
why not just `/service/{service_id}` ?

Reply:
I was wondering if it can be confused with `/service/{serviceId}.json` that is used for metadata.

Reply:
I've thought about this and this is a little bit confusing since /service/{id} returns metadata while /service/{id}/latest returns data. We should make this difference explicit someway (maybe having another endpoint ie. `service-data`)
Ideally  `/service/{id}.json` would have returned both (but it's too late now I guess)